Godnoken/Sprint-Challenge--Advanced-CSS
# Sprint Challenge: Advanced CSS

## What does this PR do?
Sprint Challenge: Advanced CSS I MVP and Stretch goals
​
## Description of tasks completed
- Answered all the questions in the README.md file
- Imported all .less files into index.less
- Implemented a complete home page with the help of mixins
- Implemented a mobile design at 500px width
- Implemented fixed navigation with opacity (Stretch)
- Implemented animations to navigation links (Stretch)
- Implemented sign in and sign up page for SpaceWalkers Magazine (Stretch)
​
## How can this be tested?
Clone the project

1. Open up README.md in your text editor of choice
2. Open up index.less and check for @import ""
3. Open up index.html and check the desktop layout. Look at index.css or .less files for code
4. Open up index.html at 500px width

Stretch
1. Open up index.html and scroll through the page vertically
2. Open up index.html and hover over the a links on the navigation bar
3. Open up index.html and click on the sign in page. To see the sign up page, click the "Register new account" link
​
## Relevant screenshot
![image](https://user-images.githubusercontent.com/17499594/66634878-4d2cf500-ec0e-11e9-917b-63bc195b1b44.png)


![image](https://user-images.githubusercontent.com/17499594/66634762-0d660d80-ec0e-11e9-9422-cdbef7f872f3.png)

![image](https://user-images.githubusercontent.com/17499594/66634769-11922b00-ec0e-11e9-9695-f5631c728e04.png)

![image](https://user-images.githubusercontent.com/17499594/66634776-1525b200-ec0e-11e9-94b4-a54760bd57e7.png)

![image](https://user-images.githubusercontent.com/17499594/66642699-8b7de080-ec1d-11e9-9efd-930c4da5f402.png)

![image](https://user-images.githubusercontent.com/17499594/66642709-90429480-ec1d-11e9-94c0-b0f2f5e8ef80.png)
